### ISSUE TITLE
Redirect Request: How to get a DS Logon account

### ISSUE THREAD

**Issue by laurwill** [department-of-veterans-affairs/va.gov-team#54951]
### Instructions
- Requests must be submitted **AT LEAST** 2 weeks in advance.  Some requests take a significant amount of time and coordination to implement, so start the process as soon as you know you will need one. 
- This issue will be used from initial request through implementation to ensure all individuals working on this are notified of status updates.  Please do not create multiple issues to track different steps.
- It is your responsibility to notify VA stakeholders as appropriate.  

### Type of request
- [X] We are retiring or taking down a page and need to redirect the URL (complete redirect section)
- [ ] We are changing the URL of an existing page (complete redirect section)
- [ ] We need a custom vanity URL (complete vanity URL section)

### Implementation date
When does this request need to be live?
Choose one:
- [ ] On a specific date, connected to a program or publishing requirement. **DATE:**
- [ ] Any day prior to a deadline/latest possible date. **DEADLINE:** 
- [ ] Any day after an earliest possible date. **EARLIEST DATE:**
- [ ] During a specific date range: ***EARLIEST DATE TO LATEST DATE:***
- [x] No hard timing requirements, publish when possible.

### Redirects 
Current URL  |  Redirect Destination or New URL
---  |  ---
[current URL](https://www.va.gov/resources/how-to-get-a-premium-ds-logon-account-online/) | [new URL](https://www.va.gov/resources/ds-logon-faqs/)

### Vanity URLs 
Vanity URL  |  Landing page
---  |  ---
requested URL | landing page URL

**Link to campaign landing page request issue:**

### Process, Roles and Responsibilities
- [X] Requesting team: Above information is provided
- [ ] Requesting team: All appropriate VA stakeholders are notified as appropriate
- [ ] IA: Request is vetted and documented and implementation plan is clear 
- [ ] IA: Request is assigned to appropriate team for implementation 
- [ ] Implementation team: Work is complete
- [ ] Implementation team: Validated in production
- [ ] Requesting team: Validates everything is correct in production and closes ticket

**Comment by jilladams** [comment]
@laurwill I updated ticket body to expose the URLs, as that makes things a little more clear for development / ensures we don't have typos, etc. Is this ready for us to implement?

**Comment by laurwill** [comment]
@jilladams thank you! I'll do that going forward. And not quite ready, I was just getting the ticket set up since I know these can take a bit of time. I'll unassign you and reassign when ready to go. Need to confirm with @jennymayoco for IA and get signoff from identity team. 

Noting reason here: VA.gov is sunsetting DS Logon as a sign-in option in September 2023. Identity team has asked us to remove pages and content that encourage Veterans to create new DS Logon accounts.

**Comment by kristinoletmuskat** [comment]
Hey @laurwill, I remember last time we talked about this you had proposed this that we redirect to login.gov content. But I'm guessing we are incrementally removing these by rerouting to the DSL FAQ page -- is that right?

![Screenshot 2023-03-10 at 2 17 51 PM](https://user-images.githubusercontent.com/122126772/224430178-62f2f0ae-93c9-4af1-9c2a-d2059f833f0a.png)

**Comment by laurwill** [comment]
Hey @kristinoletmuskat ! Yes, we previously thought this would go to the Creating an account page with info about Login.gov accounts. But then we realized that VA.gov will still need to mention DS Logon, because Veterans and service members need to use it for milConnect and TRICARE (it's a DoD credential). 

So we're now thinking that we'll maintain the DS Logon FAQs page but revise it to frame it as a DoD credential. Does this redirect make sense with that context? (And apologies, realizing I should have run this by you in that Login.gov ticket!)

**Comment by kristinoletmuskat** [comment]
Yes totally! Thank you. @jilladams we are good to go for these redirects.
